powerpacks.go

```diff
@@ -277,6 +313,30 @@ func (service *PowerpackService) Remove_number(number string, param NumberRemove
 	return
 }
 
+func (service *PowerpackService) Remove_tollfree(tollfree string, param NumberRemoveParams) (response *TollfreeDeleteResponse, err error) {
+	numberpool_path := service.Powerpack.NumberPoolUUID
+	uriSegments := strings.Split(numberpool_path, "/")
+	req, err := service.client.NewRequest("DELETE", param, "NumberPool/%s/Tollfree/%s", uriSegments[5], tollfree)
+	if err != nil {
+		return
+	}
+	response = &TollfreeDeleteResponse{}
+	err = service.client.ExecuteRequest(req, response)
+	return
+}
+
+func (service *PowerpackService) Remove_shortcode(shortcode string) (response *ShortcodeDeleteResponse, err error) {
+	numberpool_path := service.Powerpack.NumberPoolUUID
+	uriSegments := strings.Split(numberpool_path, "/")
```

Comment:
You should validate that `uriSegments` has (at least) 6 elements before accessing `uriSegments[5]`.